src/include/mutex.h

```diff
@@ -5,6 +5,8 @@
 
 #ifdef CMARK_THREADING
 
+#ifndef _WIN32
```

Comment:
I would prefer `#if defined(_POSIX_THREADS)`.  We already do use that in swift-corelibs-foundation in CoreFoundation.  That future proofs us for environments like Solaris which have multiple threading systems on a single platform.

Reply:
Should i still keep the `#else` without a condition?

Reply:
I would suggest that we be more aggressive - ensure that the else case is checking for `_WIN32`.  Falling back is always safe.